src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Services/SyntaxFacts/CSharpSyntaxFacts.cs

```diff
@@ -744,11 +744,14 @@ public bool IsIndexerMemberCRef(SyntaxNode node)
             {
                 if (useFullSpan || node.Span.Contains(position))
                 {
-                    var kind = node.Kind();
-                    if ((kind != SyntaxKind.GlobalStatement) && (kind != SyntaxKind.IncompleteMember) && (node is MemberDeclarationSyntax))
+                    if (!node.IsKind(SyntaxKind.IncompleteMember) && (node is MemberDeclarationSyntax))
```

Comment:
```suggestion
                    if (node is MemberDeclarationSyntax(not SyntaxKind.IncompleteMember))
```

does that work (i'm not actually sure).